src/unistd.rs

```diff
@@ -2136,3 +2146,368 @@ mod setres {
         Errno::result(res).map(drop)
     }
 }
+
+
+#[derive(Debug, Clone, PartialEq, Default)]
+pub struct User {
+    /// username
+    pub name: String,
```

Comment:
Why make some fields `String` if others must be `CString`?  The inconsistency is weird.

Reply:
this seems done as well since d4d5a008.